tasks/config/babel.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
       '!**/public/**',
       '!**/node_modules/**',
       '!**/bower_components/**',
-      '!**/__tests__/**'
+      '!**/__tests__/**',
+      '!**/src/forked/**'
```

Comment:
This probably seems like nitpicking, but we have to be careful with our globbing patterns.  What works today might end up accidentally matching something we don't want down the line, and when that happens the problem can be very difficult to debug.